### DISCUSSION TITLE
part-ii/2018-120/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#116] [Question comments]
Part II 2018 Probability and Measure: Paper 4, Section II, J 

https://questions.tripos.org/part-ii/2018-120/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

(e) $T$-orbits.

</details>

**Comment by DannyDuberstein** [comment]
Anyone know what the "rotation map" referred to in the examiner's report is? Seems for c) that an example with X = {0,1} works
